src/Uno.UI/UI/Xaml/Controls/ListView/ListView.cs

```diff
@@ -13,16 +13,28 @@ public ListView()
 
 		protected override bool IsItemItsOwnContainerOverride(object item)
 		{
+			global::System.Diagnostics.Debug.WriteLine("=========================================");
+			global::System.Diagnostics.Debug.WriteLine("IsItemItsOwnContainerOverride");
+			global::System.Diagnostics.Debug.WriteLine(Environment.StackTrace);
+			global::System.Diagnostics.Debug.WriteLine("=========================================");
```

Comment:
Will revert, was an attempt when I was debugging on Android where the debugger is broken 😄